crates/contract/Cargo.toml

```diff
@@ -7,13 +7,13 @@ license = "Apache-2.0"
 readme = "README.md"
 
 [dependencies]
-ink_primitives = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_metadata = { git = "https://github.com/paritytech/ink", default-features = false, features = ["derive"], optional = true }
-ink_env = { git = "https://github.com/paritytech/ink", default-features = false, features = ["ink-debug"] }
-ink_storage = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_lang = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_prelude = { git = "https://github.com/paritytech/ink", default-features = false }
-ink_eth_compatibility = { git = "https://github.com/paritytech/ink", default-features = false }
+ink_primitives = { version = "3.0.0-rc9", default-features = false }
+ink_metadata = { version = "3.0.0-rc9", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.0.0-rc9", default-features = false, features = ["ink-debug"] }
+ink_storage = { version = "3.0.0-rc9", default-features = false }
+ink_lang = { version = "3.0.0-rc9", default-features = false }
+ink_prelude = { version = "3.0.0-rc9", default-features = false }
+ink_eth_compatibility = { version = "3.0.0-rc9", default-features = false }
 
 scale = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive"] }
 scale-info = { version = "1", default-features = false, features = ["derive"], optional = true }
```

Comment:
You'll need to update these as well

Reply:
I replace the `Cargo.toml` in `./compiler` folder now by a symbolic link to `./crates/contract/Cargo.toml`, this will avoid different versions for compiler and rust analyzer in the future.